pkg/registry/consul.go

```diff
@@ -70,7 +70,7 @@ func (crl *ConsulRegistryLoad) GetCluster() (string, error) {
 	return crl.cluster, nil
 }
 
-func (crl *ConsulRegistryLoad) transfer2Url(service consul.AgentService) (common.URL, error) {
+func (crl *ConsulRegistryLoad) transfer2Url(service consul.AgentService) (*common.URL, error) {
```

Comment:
IMO, less pointer may be better

Reply:
in dubbo go v1.5.5-rc1,  common.Url was chanaged to a reference type, and there is a lock in the url , If I evaluate it (*URL), I get an error, can not pass linter

Reply:
OK